docs/api/1.x/sorting.rst

```diff
@@ -0,0 +1,19 @@
+Sorting
+#######
+
+When requesting elements (such as records) from a plural endpoint
+(such as a collection), the results can be sorted using an additional
```

Comment:
We could repeat the formulae `Plural endpoints (such as collections) support XXX`